sklearn/ensemble/iforest.py

```diff
@@ -103,6 +106,14 @@ class IsolationForest(BaseBagging):
     max_samples_ : integer
         The actual number of samples
 
+    offset_ : float
+        Offset used to define the decision function from the raw scores.
+        We have the relation: decision_function = score_samples - offset_.
+        The offset is set to 0.5 as in the original paper, except when a
+        contamination parameter different than "auto" is provided. In that
+        case, the offset is defined in such a way we obtain the expected
+        number of outliers (samples with decision function < 0) in training.
+
```

Comment:
same here for the definition of predict as decision_function >= 0 but this is not what predict does

Reply:
corrected